V2/server/test/UserTest.js

```diff
@@ -0,0 +1,67 @@
+import chai from 'chai';
+
+import chaiHttp from 'chai-http';
+
+import app from '../index';
+
+import users from '../model/users';
+
+const { expect } = chai;
+
+chai.use(chaiHttp);
+// ############ SIGNUP TEST ############
```

Comment:
Remove the comments... describe() should be the one to give a general description of the test suite

Reply:
ok, let me change that